python/tests/integration/test_methods.py

```diff
@@ -42,11 +42,14 @@ def test_crud_user(sdk: mtds.Looker40SDK):
 
     # sudo checks
     user_id = user.id
-    sdk.login_user(user_id)
+    sudo_auth = sdk.login_user(user_id)
+    assert isinstance(sudo_auth.access_token, str)
+    assert sudo_auth.access_token != ""
+    sdk.auth.login_user(user_id)
```

Comment:
and now we do this for the old behavior.

Reply:
Looks more like the other SDKs now